### DISCUSSION TITLE
Swing migration changes

### DISCUSSION THREAD

**Discussion by Misha305** [Microsoft/AADConnectConfigDocumenter#35] [Q&A]
Going from version 1.1.882.0 to 2.1.1.0 I am getting a handful of notifications in the syncrulechanges script that there are unsupported changes detected on default rules.  The Synchronization rules editor does not indicate that they are actually changed.  What criteria is the aadconectconfigDocumentor using to determin this?  Is there a list of default precedence numbers that might affect this?  I think these may be false positives in my case do to the version changes but want to be sure.

**Comment by NileshGhodekar** [comment]
Hi @Misha305 , yes you may get false positives if the default sync rules have updates to them in the newer version of AAD Connect. No way to know for the documentor as it's a "dumb" file compare utility if these changed are made by the product team or customer. It justs flags them for manual review.

**Reply by Misha305** [reply]
Thank you for the response.  After I posted this I looked at closed issues and saw that someone had the same question along with your suggestion to run the check on the old environment.  I exported the config from the 1.1.882.0 production instance and compared the config against itself.  I also checked the export config from the 2.1.1.0 instance against itself.  When comparing the same exports no unsupported changes were detected.  I think this confirms that the changes between the old and new versions are made by the product team and not my organization.  I will take a look at csexport to see if I can determine if those changes have any effect on the data I am synching.
